pub-matic-htb.js

```diff
@@ -415,9 +538,9 @@ function PubMaticHtb(configs) {
             },
             bidUnitInCents: 1, // The bid price unit (in cents) the endpoint returns, please refer to the readme for details
             lineItemType: Constants.LineItemTypes.ID_AND_SIZE,
-            callbackType: Partner.CallbackTypes.ID, // Callback type, please refer to the readme for details
+            callbackType: Partner.CallbackTypes.CALLBACK_NAME, // Callback type, please refer to the readme for details
             architecture: Partner.Architectures.SRA, // Request architecture, please refer to the readme for details
-            requestType: Partner.RequestTypes.ANY // Request type, jsonp, ajax, or any.
+            requestType: Partner.RequestTypes.JSONP // Request type, jsonp, ajax, or any.
```

Comment:
Hi, since the request method is changed to POST, the requestType needs to be changed to ajax since JSONP and POST are not compatible.

Reply:
I've changed this locally and the wrapper was able to make the following post request:
```
http://hbopenbid.pubmatic.com/translator?source=index-client
```
With following payload:
```
{"id":"1524085661221","at":1,"cur":["USD"],"imp":[{"id":"MH9ENU8G","tagId":"pubmatic_test2","bidFloor":1.75,"ext":{},"banner":{"w":300,"h":250}}],"site":{"ref":"","publisher":{"id":"301","domain":"sandbox.ht.indexexchange.com"},"domain":"sandbox.ht.indexexchange.com"},"device":{"ua":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Safari/537.36","js":1,"dnt":0,"h":800,"w":1280,"language":"en-CA","geo":{"lat":40.712775,"lon":-74.005973}},"user":{"gender":"M","geo":{"lat":40.712775,"lon":-74.005973},"yob":1982},"ext":{"wrapper":{"wp":"pbjs"}},"secure":0}
```

I'm still unable to get a 200 response from the post request.

Reply:
Changing the publisherId from 301 to 156209 should work.